src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildProject.File.cs

```diff
@@ -194,5 +194,21 @@ public void SetPropertyValue(string propertyName, string propertyValue)
             Path.IsPathFullyQualified(path)
             ? path
             : Path.Combine(projectDir, path);
+
+        public void AddImports(string item)
```

Comment:
This won't change the project file on disk. We need `AddImports` and `RemoveImports` to actually update the project file. `ProjectRoot.AddImport` should work for the add case, but I'm not sure if there's a good helper like that for removing. For that, we might need to find child nodes that match the import and use RemoveChild.

Reply:
Changed to use extension method to removelement similar to what removereferences is doing.